### PR TITLE
Add  merchandise links to pakyow.org

### PR DESCRIPTION
**Pull request by lydialowther**
screen shot 2016-07-27 at 3 29 03 pm

### CODE REVIEW

File: app/assets/css/theme.scss [hunk 1]
```diff
@@ -248,12 +248,12 @@ nav {
 }
 
 .blurbs {
-  width: 600px;
```
Comment (bryanp) [inline app/assets/css/theme.scss]: Why did this width have to be changed?

**Comment by lydialowther** [comment]
The width was changed so that the four columns could remain on one line rather then being pushed to a second line.

**Comment by bryanp** [comment]
@lydialowther It shouldn't have to be changed with the `container-3` being changed to a `container-4`. If that doesn't work perhaps there's a bug in Pakyow CSS.

**Comment by lydialowther** [comment]
You were we manually setting the width of the container and the columns. You had set the width of the container to 600px and set the columns to 200px. We increased the width of the container to the original size and and changed the column width so that they would look like you had them before.

**Comment by bryanp** [comment]
Yes, what I mean though is that sense we now want the container to be the full width, we should be able to just remove the width settings here and let it go back to its default behavior.

**Comment by lydialowther** [comment]
ah I see. I'll check that out.

**Comment by lydialowther** [comment]
Removed it and it worked.

**Pull request merged**